MetaMask/metamask-extension
# feat: 22271 add hex data test

## **Description**
This is an automation test for toggling the hex data. The user should be able to input text via the send transaction feature.



## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/22271

## **Manual testing steps**

1. Go to User settings page
2. In the advanced tab, there should be toggle for "Show Hex Data"
3. In the send transaction, verify that the HexData text box appears.

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
